src/ADAL.PCL.Android/BrokerProxy.cs

```diff
@@ -430,6 +430,13 @@ private Bundle GetBrokerOptions(AuthenticationRequest request)
                 request.Version);
             brokerOptions.PutString(BrokerConstants.AccountExtraQueryParam,
                 request.ExtraQueryParamsAuthentication);
+
+            if(request.Claims != null)
+            {
+                brokerOptions.PutString(BrokerConstants.SkipCache, Boolean.TrueString.ToLower());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define the locale to be used in this string operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS1449)